docs/chipflow-toml-guide.rst

```diff
@@ -139,19 +140,37 @@ Available pad rings
 +----------+-----------+--------------------+------------------------------------+
 
 
-``[silicon.pads]``
-------------------
+``[silicon.power]``
+-------------------
+
+This section outlines the connection of pads to the power supply available for the selected process and package.
+These pads are declared with the :term:type parameter, along with a name and other optional information, like voltage.
 
-The ``silicon.pads`` section lists special pads. In general you are unlikely to need to add to this.
-Each pad specified with the name used by the design and two parameters: :term:`type` and :term:`loc`.
+Note that in this context, the :term:type parameter can only be ``ground`` or ``power``.
 
-.. code-block:: TOML
+The package definition provides default locations for standard pins like power, ground, clocks, and resets. You only need to specify the name and properties.
+
+[chipflow.silicon.power]
+vdd = { type = "power", name = "vdd", voltage = "1.8V" }
```

Comment:
it'd be nice to have an example of separate core and IO power here, as that will always almost be the case